### PR TITLE
Update to 2.0.1-preview1

### PR DESCRIPTION
**Pull request by wtgodbe**
CC @weshaggard

### CODE REVIEW

File: dependencies.props [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <!-- Full package version strings that are used in other parts of the build. -->
   <PropertyGroup>
-    <CoreClrPackageVersion>2.0.0</CoreClrPackageVersion>
+    <CoreClrPackageVersion>2.0.1</CoreClrPackageVersion>
```
Comment (weshaggard) [inline dependencies.props]: This isn't the version you want to bump. it is the PackageVersion property above.

File: dependencies.props [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```
Comment (weshaggard) [inline dependencies.props]: You will need to set StabilizePackageVersion to false.

File: dependencies.props [hunk 3]
```diff
@@ -8,7 +8,7 @@
...
     <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == ''">$(PackageVersion)</StableVersion>
 
-    <PreReleaseLabel>preview3</PreReleaseLabel>
+    <PreReleaseLabel>preview1</PreReleaseLabel>
```
Comment (weshaggard) [inline dependencies.props]: Please change this to servicing instead of preview1.

**Comment by wtgodbe** [comment]
@weshaggard updated

**Pull request merged**